sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.institutions.dto.ts

```diff
@@ -46,6 +46,8 @@ export class InProgressApplicationOfferingChangesAPIOutDTO {
   studyEndDate: string;
   fullName: string;
   status: ApplicationOfferingChangeRequestStatus;
+  createdAt: string;
```

Comment:
IMO, I think we should create 2 DTO for each controller there are some differences in some fields or at least extend

Reply:
Yes I agree, the DTO's are used by 2 different users and we should have an abstraction between them.